packages/atomic/package.json

```diff
@@ -34,9 +34,9 @@
     "test": "stencil test --spec",
     "test:watch": "stencil test --spec --watchAll",
     "generate": "stencil generate",
-    "cypress:test": "./node_modules/cypress/bin/cypress run --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser chrome",
-    "cypress:test:firefox": "./node_modules/cypress/bin/cypress --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser firefox",
-    "cypress:open": "./node_modules/cypress/bin/cypress open",
+    "cypress:test": "cypress run --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser chrome",
+    "cypress:test:firefox": "cypress --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser firefox",
+    "cypress:open": "cypress open",
```

Comment:
I adjusted these commands because they error on my machine otherwise,

![current-cypress-command-error](https://user-images.githubusercontent.com/5565841/149033185-203d16a6-dd1b-439d-8f5d-f87e9d534e0c.PNG)

Reply:
Hmm the full path were used for a reason, perhaps jenkins. Now that we are on github, might not be a problem